Source/AliveLibAE/BaseGameObject.hpp

```diff
@@ -6,6 +6,28 @@
 
 ALIVE_VAR_EXTERN(unsigned int, sGnFrame_5C1B84);
 
+struct Timer
+{
+    int mTimer;
+
+    // Helper to check if a timer has expired
+    inline bool Expired()
+    {
+        return static_cast<int>(sGnFrame_5C1B84) > mTimer;
+    }
+
+    inline bool Expired2()
```

Comment:
What is the functional diff between these? Is one Expiring and one Expired ?

Reply:
i dont really think they are different except the +-1 frame depending on the condition. probably just inconsistent OWI code i guess